frontend/src/pages/Frontpage/FrontpageContainer.jsx

```diff
@@ -25,9 +36,31 @@ const FrontpageContainer = ({ children }) => {
     );
   };
 
+  const changeSubThread = async event => {
+    if (event === 'All') {
+      setPostsToDisplay(retrievedPosts.reverse());
+    } else {
+      const response = await getPostsForSubthread(event);
+
+      if (!Array.isArray(response) || !response.length) {
+        setPostsToDisplay([]);
+      } else {
+        setPostsToDisplay([]);
```

Comment:
`setPostsToDisplay(response);` The posts are not getting displayed.

Reply:
Thank you for spotting that, I have made the change